src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/InternetAddress.kt

```diff
@@ -3,22 +3,26 @@ package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
 import jakarta.persistence.Column
 import jakarta.persistence.DiscriminatorColumn
 import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
 import jakarta.persistence.Id
 import jakarta.persistence.Inheritance
+import jakarta.persistence.SequenceGenerator
 import jakarta.persistence.Table
 
 @Entity
 @Table(name = "INTERNET_ADDRESSES")
 @DiscriminatorColumn(name = "OWNER_CLASS")
 @Inheritance
-abstract class InternetAddress {
+abstract class InternetAddress(
+  @Column(name = "INTERNET_ADDRESS")
+  open val internetAddress: String,
+
+  @Column(name = "INTERNET_ADDRESS_CLASS")
+  open val internetAddressClass: String,
```

Comment:
decided not to map this a reference data since Person only ever have email addresses